kits/token-wrapper/contracts/TokenWrapper.sol

```diff
@@ -0,0 +1,143 @@
+        require(_token.controller() == address(this), ERROR_INVALID_TOKEN_CONTROLLER);
+
+        token = _token;
+        erc20 = _erc20;
+        token.enableTransfers(false);
+    }
+
+    /**
+     * @notice Lock `_amount` tokens
+     */
+    function lock(uint256 _amount) external {
+        require(_amount > 0, ERROR_LOCK_AMOUNT_ZERO);
+
+        lockedAmount[msg.sender] = lockedAmount[msg.sender].add(_amount);
+        emit TokensLocked(msg.sender, _amount);
+
+        require(erc20.transferFrom(msg.sender, address(this), _amount), ERROR_ERC20_TRANSFER_FROM_FAILED);
+        require(token.generateTokens(msg.sender, _amount), ERROR_TOKEN_MINT_FAILED);
+    }
+
+    /**
+     * @notice Unlock `_amount` tokens
+     */
+    function unlock(uint256 _amount) external {
+        require(_amount <= lockedAmount[msg.sender], ERROR_INVALID_UNLOCK_AMOUNT);
+
+        lockedAmount[msg.sender] = lockedAmount[msg.sender].sub(_amount);
+        emit TokensUnlocked(msg.sender, _amount);
+
+        require(token.destroyTokens(msg.sender, _amount), ERROR_TOKEN_BURN_FAILED);
+        require(erc20.transfer(msg.sender, _amount), ERROR_ERC20_TRANSFER_FAILED);
+    }
+
+    /**
+     * @notice Give control of the MiniMe token over to `_newController`
+     * @param _newController Ethereum address of contract that will control the token
+     */
+    function changeTokenController(address _newController) external auth(CONTROLLER_MANAGER_ROLE) {
+        token.changeController(_newController);
+    }
+
+    /*
+    * @dev Notifies the controller about a token transfer allowing the controller to react if desired
+    * @return False aways
+    */
+    function onTransfer(address, address, uint256) external onlyToken returns (bool) {
+        return false;
+    }
+
+    /**
+     * @dev Notifies the controller about an approval allowing the controller to react if desired
+     * @return False always
+     */
+    function onApprove(address, address, uint256) external onlyToken returns (bool) {
+        return false;
+    }
+
+    /**
+     * @dev Notifies the controller when the token contract receives ETH allowing the controller to react if desired
+     * @return False always
+     */
+    function proxyPayment(address) external payable onlyToken returns (bool) {
+        return false;
+    }
+
+    /**
+     * @notice Execute desired action as a token holder
+     * @dev IForwarder interface conformance. Forwards any token holder action.
+     * @param _evmScript Script being executed
+     */
+    function forward(bytes _evmScript) public {
+        require(canForward(msg.sender, _evmScript), ERROR_CAN_NOT_FORWARD);
+        bytes memory input = new bytes(0);
+
+        // Add the managed token to the blacklist to disallow a token holder from executing actions
+        // on the token controller's (this contract) behalf
+        address[] memory blacklist = new address[](2);
+        blacklist[0] = address(token);
+        blacklist[1] = address(erc20);
```

Comment:
Why this one too? What would be the danger?

Reply:
For consistency, since we are not allowing holders to interact with the minime token in the script, I think it makes sense to do the same for the wrapped one.